### PR TITLE
Fixed point `div_pipe`s. Use standardized naming convention. Signed values for `exp` generator.

### PR DESCRIPTION
**Pull request by cgyurgyik**
Also updates the `exp` generator to support negative `x` values by taking the reciprocal, i.e.
```
e^-x = 1/e^x
```

### CODE REVIEW

File: tests/correctness/numeric-types/fixed-point/binary-operators.expect [hunk 1]
```diff
@@ -8,6 +8,12 @@
   "signed_add": [
     "0.5"
   ],
+  "signed_div": [
+    "-0.5"
```
Comment (rachitnigam) [inline tests/correctness/numeric-types/fixed-point/binary-operators.expect:12]: Why are the values for signed and unsigned `div` different? Are the bits in the fixed point representations being interpreted differently?

Comment (cgyurgyik) [reply tests/correctness/numeric-types/fixed-point/binary-operators.expect:12]: For a sanity check, I use different values for signed and unsigned binary operators:

https://github.com/cucapra/calyx/blob/4862208056e54118eafc472abc068a44279f1b97/tests/correctness/numeric-types/fixed-point/binary-operators.futil.data#L1-L45

Comment (rachitnigam) [reply tests/correctness/numeric-types/fixed-point/binary-operators.expect:12]: Got it!

**Comment by rachitnigam** [comment]
LGTM! And fixed point hype!!

**Comment by cgyurgyik** [comment]
> LGTM! And fixed point hype!!

I need to add a `if` statement for the `exp` generator so that it only negates/takes the reciprocal if `x` is negative. I'll push when I'm done.

**Pull request merged**